<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="cat_orange">
    <link name="body">
      <pose>0 0 0 0 0 0</pose>
      <collision name="collision">
        <geometry>
            <box>
              <size>0.32 0.1 0.14</size>
            </box>
        </geometry>
        <pose>0 0 0.2 0 0 0</pose>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://cat_orange/meshes/Cat_v1_l3.obj</uri>
            <scale>0.01 0.01 0.01</scale>
          </mesh>
        </geometry>
        <pose>0 0 0 0 0 1.570796327</pose>
      </visual>
      <inertial>
        <inertia>
          <ixx>0.009867</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.04067</iyy>
          <iyz>0</iyz>
          <izz>0.03747</izz>
        </inertia>
        <mass>4</mass>
        <pose>0 0 0.2 0 0 0</pose>
      </inertial>
    </link>
    <joint name="head_joint" type="fixed">
      <parent>body</parent>
      <child>head</child>
    </joint>
    <link name="head">
      <collision name="collision">
        <geometry>
            <box>
              <size>0.1 0.1 0.08</size>
            </box>
        </geometry>
      </collision>
      <inertial>
        <inertia>
          <ixx>0.0003417</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.0003417</iyy>
          <iyz>0</iyz>
          <izz>0.0004167</izz>
        </inertia>
        <mass>0.25</mass>
      </inertial>
      <pose>0.15 0 0.3 0 0 0</pose>
    </link>
    <joint name="front_legs_joint" type="fixed">
      <parent>body</parent>
      <child>front_legs</child>
    </joint>
    <link name="front_legs">
      <collision name="collision">
        <geometry>
            <box>
              <size>0.03 0.1 0.14</size>
            </box>
        </geometry>
      </collision>
      <inertial>
        <inertia>
          <ixx>0.0006167</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.0004417</iyy>
          <iyz>0</iyz>
          <izz>0.00024167</izz>
        </inertia>
        <mass>0.25</mass>
      </inertial>
      <pose>0.1 0 0.07 0 0 0</pose>
    </link>
    <joint name="rear_legs_joint" type="fixed">
      <parent>body</parent>
      <child>rear_legs</child>
    </joint>
    <link name="rear_legs">
      <collision name="collision">
        <geometry>
            <box>
              <size>0.04 0.1 0.14</size>
            </box>
        </geometry>
      </collision>
      <inertial>
        <inertia>
          <ixx>0.0006167</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.0004417</iyy>
          <iyz>0</iyz>
          <izz>0.00024167</izz>
        </inertia>
        <mass>0.25</mass>
      </inertial>
      <pose>-0.14 0 0.07 0 0 0</pose>
    </link>
  </model>
</sdf>
